base_classes/NXdetector.nxdl.xml

```diff
@@ -44,6 +44,7 @@
     <symbol name="nP"><doc>number of scan points (only present in scanning measurements)</doc></symbol>
     <symbol name="i"><doc>number of detector pixels in the first (slowest) direction</doc></symbol>
     <symbol name="j"><doc>number of detector pixels in the second (faster) direction</doc></symbol>
+    <symbol name="k"><doc>number of detector pixels in the third (if necessary, fastest) direction</doc></symbol>
     <symbol name="tof"><doc>number of bins in the time-of-flight histogram</doc></symbol>
```

Comment:
```suggestion
    <symbol name="nP"><doc>Number of scan points (only present in scanning measurements)</doc></symbol>
    <symbol name="i"><doc>Number of detector pixels in the first (slowest) direction</doc></symbol>
    <symbol name="j"><doc>Number of detector pixels in the second (faster) direction</doc></symbol>
    <symbol name="k"><doc>Number of detector pixels in the third (if necessary, fastest) direction</doc></symbol>
    <symbol name="tof"><doc>Number of bins in the time-of-flight histogram</doc></symbol>
```

Reply:
this has not been discussed/approved in the NIAC meeting, so I am not touching it